dotcom-rendering/package.json

```diff
@@ -28,6 +28,7 @@
     "cypress:run": "cypress run",
     "cypress:open:e2e": "cypress open --e2e --browser=chrome",
     "cypress:run:e2e": "cypress run --spec 'cypress/e2e/**/*'",
+    "cypress:run:percy": "percy exec -- cypress run --spec 'cypress/e2e/**/snapshot-*'",
```

Comment:
`visual` works. I do also see how `snapshot` works as well...

Lets go snapshot, I sense that this could be my own personal perspective tainting things and maybe snapshot is the best option overall.

Reply:
`visual` is growing on me as ye olde Cypress also used to have a `snapshot` plugin.

But I will merge as is and it can be updated once a wider consensus forms.